compose/parallel.py

```diff
@@ -255,26 +258,60 @@ def write(self, obj_index, status):
         self.stream.flush()
 
 
-def parallel_operation(containers, operation, options, message):
+class ParallelStreamWriterNoANSI(object):
```

Comment:
There's a lot of code duplication happening here that we can probably avoid by adding a `noansi` parameter and a handful of conditionals to the original class.